index.js

```diff
@@ -1,5 +1,19 @@
 function scoreCalculator(arr) {
-  // do work here
+  if (!Array.isArray(arr)) {
+    throw new Error('input needs to be an array!');
+  }
+
+  let score = 0;
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] < 5) {
+      score += 10;
+    }
+    if (arr[i] >= 5 && arr[i] <= 10) {
+      score += 5;
+    }
+  }
+
+  return score;
```

Comment:
This doesn't handle the bonus case where if all the values are below 5, then award an extra 100 points.